Decred Wallet/Features/Overview/OverviewViewController.swift

```diff
@@ -118,6 +126,16 @@ class OverviewViewController: UIViewController {
         MultiWalletSyncDetailsLoader.setup(for: self.multipleWalletsSyncDetailsTableView)
     }
     
+    @objc func refreshRecentActivityAndUpdateBalance() {
+        self.refreshControl.beginRefreshing()
```

Comment:
TransactionHistory page uses `self.refreshControl.showLoader(in: tableView)` instead of `self.refreshControl.beginRefreshing()`.

If the previous suggestion of adding the refresh control to the recent activity tableview works, then this should become
```
self.recentTxTableRefreshControl.show(in: self.recentTransactionsTableView)
```